pkg/apis/core/validation/validation_test.go

```diff
@@ -9608,6 +9642,26 @@ func TestValidatePod(t *testing.T) {
 				Spec: validPodSpec(nil),
 			},
 		},
+		"AppArmor profile annotation must apply to a container": {
+			expectedError: "metadata.annotations[container.apparmor.security.beta.kubernetes.io/fake-ctr]",
```

Comment:
We'd change this again for GA, right?

Reply:
This looks unneeded, as pod or container fields wouldn't have this problem, where annotations do.